packages/sdk-core/src/main/kotlin/ly.iterative.itly/PluginBase.kt

```diff
@@ -9,18 +9,24 @@ abstract class PluginBase(
         return id
     }
     override fun load(options: Options) {}
+
+    override fun validate(event: Event): ValidationResponse? {
+        return null
```

Comment:
Not sure about this but agree it sucks to have to create new objects all the time.